test/core.js

```diff
@@ -615,6 +619,16 @@ describe('snabbdom', function() {
           assert.deepEqual(map(inner, elm.children), arr.filter(function(x) {return x != null;}));
         }
       });
+      it('accepts symbol as key', function() {
+        var vnode1 = h('span', [Symbol()].map(spanNum));
+        var vnode2 = h('span', [Symbol('1'), Symbol('2'), Symbol('3')].map(spanNum));
+        elm = patch(vnode0, vnode1).elm;
+        assert.equal(elm.children.length, 1);
```

Comment:
I understand now! but still this test is not enough! if what we want to test is that key could be a symbol, then we need at least 2 tests:

1. test that patching two similar vnodes with the same symbol as a key, reuses the element
2. test that patching two similar vnodes with different symbol as key, does not reuse the element